cloud/azure/versions.tf

```diff
@@ -4,6 +4,14 @@ terraform {
       source  = "splunk-terraform/signalfx"
       version = ">= 4.26.4"
     }
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = ">= 1.44"
```

Comment:
If this is for the next "major" version of this module, would be better to use `>= 2.0` for azure, 1.x is now quite legacy/deprecated

Reply:
done